src/crud.js

```diff
@@ -0,0 +1,60 @@
+const addButton = document.querySelector('.add-button');
```

Comment:
Make sure to fix when I edit task to save itself, but when I reload the page and add new task it goes back to the previous tasks name